python/tests/api/writer/test_whylabs.py

```diff
@@ -51,14 +51,20 @@ def teardown_method(self) -> None:
         SessionManager.reset()
         del os.environ[EnvVariableName.WHYLOGS_CONFIG_PATH.value]
 
-    def test_writer_throws_for_anon_sessions(self) -> None:
+    @pytest.fixture
+    def results(self, pandas_dataframe):
+        return why.log(pandas=pandas_dataframe)
+
+    def test_writer_throws_for_anon_sessions(self, results) -> None:
+        # NOTE: Assumes WHYLABS_API_KEY environment variable is not set
```

Comment:
Can you add an assert to check that assumption?

Reply:
I added code to unset it.